launcher-service/src/controllers/apicontroller.ts

```diff
@@ -20,6 +20,7 @@ export const createJobCreator = async (
         email,
         password,
         address,
+        isAdmin,
```

Comment:
This does not seem good because anyone may be an administrator in this manner. We can easily pass true from the frontend and then the person has access to the admin privileges.

Let it be for the time being, but it should definitely go into tech debt.